worker/__init__.py

```diff
@@ -71,18 +88,20 @@ def main(event: func.EventGridEvent, outputEvent: func.Out[func.EventGridOutputE
                 "actions": msg['actions'],
             },
             "response": resp,
+            "services": resp.get("services", None),
+            "updatetag": resp.get("updatetag", None),
         }
 
-        if msg.get('inventoryRefresh'):

```

Comment:
similar to AWS, we should have 2 conditions here.
1. if services has value, push it to requestTopic
2. else push it to resultTopic